### PR TITLE
KAFKA-15051: add missing GET plugin/config endpoint

### PR DESCRIPTION
**Pull request by jeqo**

### CODE REVIEW

File: docs/connect.html [hunk 1]
```diff
@@ -320,6 +320,7 @@ <h4><a id="connect_rest" href="#connect_rest">REST API</a></h4>
 
     <ul>
         <li><code>GET /connector-plugins</code>- return a list of connector plugins installed in the Kafka Connect cluster. Note that the API only checks for connectors on the worker that handles the request, which means you may see inconsistent results, especially during a rolling upgrade if you add new connector jars</li>
+        <li><code>GET /connector-plugins/{connector-type}/config</code> - get the configuration definition for the specified connector plugin.</li>
```
Comment (C0urante) [inline docs/connect.html]: This endpoint works on all plugin types, not just connectors, right? Should we be try to be clearer about that?

```suggestion
        <li><code>GET /connector-plugins/{plugin-type}/config</code> - get the configuration definition for the specified plugin.</li>
```

**Comment by vvcephei** [comment]
Retest this

**Comment by vvcephei** [comment]
test this please

**Comment by C0urante** [comment]
@vvcephei I kicked off another Jenkins build--I haven't seen GitHub comments working for a while to re-trigger them. Are you trying to test that out?

**Comment by vvcephei** [comment]
Yeah, sorry for the weird drive-bys. I was indeed trying to see if those commands still work. Apparently, they do not.

I think this is the plugin documentation, but it also looks like it was associated with security flaws, so maybe Infra disabled it: https://plugins.jenkins.io/ghprb/

I'll raise a ticket with them. At the least, if it doesn't work anymore, they should remove the docs for that role in asf.yaml.

**Review by C0urante — APPROVED** [review]
LGTM, thanks @jeqo!

**Comment by C0urante** [comment]
@vvcephei haha, no worries! I'll leave Jenkins alone if I see you in the neighborhood testing again 🙏

**Pull request merged**